.zuul.yaml

```
@@ -0,0 +1,5 @@
+- project:
+    check:
+      jobs:
+        - noop
```

Let's dedent this to comply w/ [the yamllint config](https://github.com/cherrypy/cheroot/blob/master/.yamllint#L2)
```suggestion
      - noop
```